### ISSUE TITLE
Train error due to detach

### ISSUE THREAD

**Issue by dister520** [shleecs/DeRaindrop_unofficial#2]
I am trying to train and am seeing the following error.
 File "/home/bhaktivideo/Downloads/DeRaindrop_unofficial-master/train_module.py", line 139, in train_start
    loss_G, loss_D, loss_PL, loss_ML, loss_att, loss_MAP, MSE_loss= self.forward_process(I_,GT_)
  File "/home/bhaktivideo/Downloads/DeRaindrop_unofficial-master/train_module.py", line 87, in forward_process
    loss_PL = self.criterionPL(O_, GT_.detach())

I did add autograd to the system. Any advice thank you?

**Comment by Yang-Kaixing** [comment]
i meets the same problem as well, could you please share your solutions?
